### ISSUE TITLE
DESCRIBE doesn't work with Avro Decimal

### ISSUE THREAD

**Issue by millin** [confluentinc/ksql#3059]
KSQL ver: master branch (commit 0346933)

I have Avro topic with Decimal field that declared in schema as:
```
{
  "name": "coefficient",
  "type": [
    {
      "type": "bytes",
      "scale": 1,
      "precision": 4,
      "connect.version": 1,
      "connect.parameters": {
        "scale": "1",
        "connect.decimal.precision": "4"
      },
      "connect.default": "AA==",
      "connect.name": "org.apache.kafka.connect.data.Decimal",
      "logicalType": "decimal"
    },
    "null"
  ],
  "default": "\u0000"
}
```
I created STREAM with Avro formal:
```
CREATE STREAM AVRO_WITH_DECIMAL
 WITH (KAFKA_TOPIC='decimal_test', VALUE_FORMAT='AVRO');
```

But KSQL can't DESCRIBE it:
```
ksql> DESCRIBE AVRO_WITH_DECIMAL;
Invalid type in schema: BYTES
```

SELECT * FROM AVRO_WITH_DECIMAL
works fine

**Comment by big-andy-coates** [comment]
Thanks reporting @millin - I'm not sure the DECIMAL work is actually complete yet. So you may be running into issues due to partial implementation.  I'll raise this though.

**Comment by agavra** [comment]
fixed by #3083
